script.js

```diff
@@ -1,292 +1,314 @@
+  document.querySelector('.pop-img').src = postDetails[3].img;
+  document.querySelector('.pop-txt').innerHTML = postDetails[3].description;
+  document.querySelector('#live').href = postDetails[3].live;
+  document.querySelector('#source').href = postDetails[3].source;
+});
+
+const pro4 = document.querySelector('#pro4');
+
+pro4.addEventListener('click', () => {
+  popupshow();
+  document.querySelector('.pop-title').innerHTML = postDetails[4].title;
+  document.querySelector('.pop-img').src = postDetails[4].img;
+  document.querySelector('.pop-txt').innerHTML = postDetails[4].description;
+  document.querySelector('#live').href = postDetails[4].live;
+  document.querySelector('#source').href = postDetails[4].source;
+});
+
+const pro5 = document.querySelector('#pro5');
+
+pro5.addEventListener('click', () => {
+  popupshow();
+  document.querySelector('.pop-title').innerHTML = postDetails[5].title;
+  document.querySelector('.pop-img').src = postDetails[5].img;
+  document.querySelector('.pop-txt').innerHTML = postDetails[5].description;
+  document.querySelector('#live').href = postDetails[5].live;
+  document.querySelector('#source').href = postDetails[5].source;
+});
+
+const pro6 = document.querySelector('#pro6');
+
+pro6.addEventListener('click', () => {
+  popupshow();
+  document.querySelector('.pop-title').innerHTML = postDetails[6].title;
+  document.querySelector('.pop-img').src = postDetails[6].img;
+  document.querySelector('.pop-txt').innerHTML = postDetails[6].description;
+  document.querySelector('#live').href = postDetails[6].live;
+  document.querySelector('#source').href = postDetails[6].source;
+});
+
+const closeButton = document.querySelector('.close');
+closeButton.addEventListener('click', () => {
+  popup.classList.remove('active');
+  overlay.classList.remove('active');
+});
+
+const email = document.querySelector('#email');
+const form = document.querySelector('.contact-us');
+const error = email.nextElementSibling;
+
+const inputPatern = /^[a-z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-z0-9-]+(?:\.[a-z0-9-]+)*$/;
+
+form.addEventListener('submit', (event) => {
+  const valid = inputPatern.test(email.value);
+  if (!valid) {
+    error.style.height = '50px';
+    error.style.padding = '10px';
+    error.innerHTML = '<i class="fa-solid fa-triangle-exclamation"></i>  Please make sure you enter your email in lowercase!';
+    event.preventDefault();
+  } else {
+    error.innerHTML = '';
+  }
+});
+
+const inputFields = document.querySelectorAll('input, textarea');
+
+function save() {
+  const data = {};
+  // eslint-disable-next-line no-restricted-syntax
+  for (const inputField of inputFields) {
+    data[inputField.name] = inputField.value;
+  }
+  localStorage.setItem('data', JSON.stringify(data));
+}
+
+function loadData() {
+  const data = JSON.parse(localStorage.getItem('data'));
+  if (data) {
+    // eslint-disable-next-line no-restricted-syntax
+    for (const inputField of inputFields) {
```

Comment:
- [ ] As I suggested above, you are not meant to use any `linters disable rule` on your code unless otherwise stated, I'll suggest you remove the linters disable and try to fix the issue. Following the suggestion I gave above will also resolve this issue. :smile: :+1:

Reply:
Thank you for your valuable feedback @V-Blaze ❤️